go.mod

```diff
@@ -39,3 +40,5 @@ require (
 	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 	k8s.io/klog v0.4.0
 )
+
+go 1.13
```

Comment:
we're still supporting 1.12 for now, what does this line do?

Reply:
That's the line my VS Code Go extension keeps adding. Until I find and disable the setting that controls it, I'll be careful to delete manually.